hickle/tests/test_08_bitshuffle.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
That puzzled me, using numeric codes in a test name.  At Seti BL, we use the ```pytest-order``` package to ensure ordering of tests.

E.g. ```@pytest.mark.order(1)``` right before a test function ensures that it runs first.

https://pytest-ordering.readthedocs.io/en/develop/

Reply:
Is an alternative option the thing is that this would mean to mark a lot of functions and to keep track that when new test functions have to be added to an existing test set `test_*.py` are properly numbered, and that all following are also properly renumbered. Which is quite an effort and can go wrong. Further pytests has a bit a strange sense of ordering tests. The test files `test_*.py` are ordered and loaded in lexicographic order while the test functions inside are than collected and executed in the order they are defined. Luckily hickle relies on pytest only otherwise things would even worse. So the number in the file name enforce that pytest loads the `test_*.py` modules in a predefined order while the functions inside are defined in the order they should or have to be executed with the effect that there is no need for an extra package and no need to keep of proper order of individual test function. 

However. Do you happen to know whether `pytest-order` package would also allow to just define the order of the `test_*.py` files, while the order of the test functions inside a file would stay the default `as defined`, `as encountered` without the need to append an ordering mark to each and every function. In that case it could be worth considering.